packages/metadata-hash/src/get-type-tree.ts

```diff
@@ -0,0 +1,121 @@
+import { Lookup, LookupValue, ScalePrimitive, TypeDef, TypeRef } from "./codecs"
+
+const bitSequenceBytes = {
+  u8: 1,
+  u16: 2,
+  u32: 4,
+  u64: 8,
+  u128: 16,
+  u256: 32,
```

Comment:
Bitsequence only goes up to u64. https://github.com/bkchr/merkleized-metadata/blob/18c9f872ce583f9aaf53a25717af6f3db7605871/src/extrinsic_decoder.rs#L129-L134
```suggestion
```